package.json

```diff
@@ -19,14 +19,13 @@
     "test-backend": "env-cmd -f tests/.test-env jest --coverage --maxWorkers=4",
     "test-backend:watch": "env-cmd -f tests/.test-env jest --watch",
     "test:frontend": "npm --prefix frontend test",
-    "build": "npm run build-backend && npm run build-frontend",
-    "build-backend": "tsc -p tsconfig.build.json",
-    "build-frontend": "webpack --config webpack.prod.js",
-    "build-frontend-dev": "webpack --config webpack.dev.js",
-    "build-frontend-dev:watch": "webpack --config webpack.dev.js --watch",
+    "build": "npm run build:backend && npm run build:frontend",
+    "build:frontend": "npm run --prefix frontend build",
+    "build:backend": "tsc -p tsconfig.build.json",
     "start": "node -r dotenv/config dist/backend/src/app/server.js",
     "dev": "docker-compose up --build",
-    "docker-dev": "npm run build-frontend-dev:watch & ts-node-dev --respawn --transpile-only --inspect=0.0.0.0 --exit-child -r dotenv/config -- src/app/server.ts",
+    "dev:backend": "tsnd --respawn --transpile-only --inspect=0.0.0.0 --exit-child -- src/app/server.ts",
```

Comment:
are the `dotenv` options no longer required?

Reply:
Thanks for the catch, probably missed from the merging of develop back into this branch